modules/ROOT/pages/amazon/amazon-sns-connector-design-center.adoc

```diff
@@ -0,0 +1,61 @@
+= Amazon SNS Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Anypoint Design Center's Flow Designer enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the
+xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Configure the target component for your app.
+
+For information on the connector field values, see
+the xref:amazon/amazon-sns-connector-reference.adoc[Amazon SNS Connector Reference].
+
+== Configure the Input Source Trigger
+
+A trigger starts your app. You can use the `HTTP Listener` operation to start your app from a browser, use a command such as `curl`, or use the `Scheduler` to start your app on a timed schedule.
+
+. In Design Center, click *Create*.
+. Click *Create new application* to open Flow Designer.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
+. Click on the trigger card.
+. If you are using the `HTTP Listener` operation as a trigger, search for and select *HTTP Listener*.
+. In the HTTP Listener's *General* tab, enter the path-absolute-URL that defines the resource location.
+. Click off the trigger card.
+
+Design Center automatically saves all changes.
+
+NOTE: Amazon SNS Connector does not provide trigger operations.
+
+== Configure the Target Component
+
+The target component enables your app to process data received from the trigger. The Amazon SNS connector provides operations that help you integrate Amazon SNS with third party applications and services. For example, this integration enables you to create connectivity and deliver notifications and messages to SQS queues and HTTP endpoints.
+
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the connector name.
+. Select the connector name.
+. Select an operation for the connector, such as *Get Topic Attributes*.
+. If needed, specify values that are required in the *General* tab. For the *Get Topic Attributes* operation, enter the Amazon resource name whose properties you want to get in the *Topic Arn* field.
+. Select *Click here to set it up* to set up other required fields.
++
+For the *Get Topic Attributes* operation, the required fields are:
++
+[%header,cols="30s,70a",width=80%]
+|===
+|Parameter |Description
+|*Configuration Name* |Name used to reference the configuration .
```

Comment:
No item in this text should end with a period, because there is not a single sentence in that group.
```suggestion
|*Configuration Name* |Name used to reference the configuration
```